workspace/src/app/views/shared/RuleEditor/view/sidebar/RuleEditorOperatorSidebar.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { RuleEditorContext } from "../../contexts/RuleEditorContext";
+import { Grid, GridColumn, GridRow, Spacing } from "gui-elements";
+import Loading from "../../../Loading";
+import { RuleOperatorList } from "./RuleOperatorList";
+import { IRuleOperator } from "../../RuleEditor.typings";
+import { extractSearchWords, matchesAllWords } from "gui-elements/src/components/Typography/Highlighter";
+import { SidebarSearchField } from "./SidebarSearchField";
+import { partitionArray, sortLexically } from "../../../../../utils/basicUtils";
+
+/** Contains the list of operators that can be dragged and dropped onto the editor canvas and supports filtering. */
+export const RuleEditorOperatorSidebar = () => {
+    const editorContext = React.useContext(RuleEditorContext);
+    const [filteredOperators, setFilteredOperators] = React.useState<IRuleOperator[]>([]);
+    // The query that was input in the search field. This won't get immediately active.
+    const [textQuery, setTextQuery] = React.useState<string>("");
+    const searchWords = extractSearchWords(textQuery);
+
+    // Filter operator list when active query or filters change
+    React.useEffect(() => {
+        if (editorContext.operatorList) {
+            if (searchWords.length > 0) {
+                setFilteredOperators(filterAndSortOperators(editorContext.operatorList, searchWords));
+            } else {
+                setFilteredOperators(editorContext.operatorList);
+            }
+        }
+    }, [textQuery, editorContext.operatorList]);
+
+    return editorContext.operatorListLoading ? (
+        <Loading />
+    ) : (
+        <Grid data-test-id={"rule-editor-sidebar"} verticalStretchable={true} useAbsoluteSpace={true}>
+            <GridRow>
+                <GridColumn full style={{ paddingTop: "3px" }}>
```

Comment:
Why using the padding styles instead of a horizontal Spacing? The padding value then is not managed by our style configuration?